features/admin/invite_supplier_contributor.feature

```diff
@@ -27,10 +27,40 @@ Scenario Outline: Prohibited user roles cannot manage supplier users
     | admin-ccs-sourcing      |
     | admin-manager           |
 
+@skip-staging
+Scenario Outline: Prohibited user roles cannot invite users to a supplier
+  Given I am logged in as the existing <role> user
+  When I visit the /admin/suppliers?supplier_name=DM+Functional+Test+Supplier+-+Invite+a+contributor+feature page
+  And I click the summary table 'Users' link for the 'DM Functional Test Supplier - Invite a contributor feature' link
+  Then I don't see the 'Send invitation' button
+
+  Examples:
+    | role                      |
+    | admin-framework-manager   |
+    | admin-ccs-category        |
+    | admin-ccs-data-controller |
+
+@requires-credentials @notify @skip-preview @skip-local
+Scenario Outline: Correct users can invite a contributors to a supplier account
+  Given I am logged in as the existing <role> user
+  And I have a supplier with:
+    | name          | DM Functional Test Supplier - Invite a contributor feature |
+  And I click the 'Edit supplier accounts or view services' link
+  And I enter 'DM Functional Test Supplier - Invite a contributor feature' in the 'supplier_name' field and click its associated 'Search' button
+  And I click a summary table 'Users' link for 'DM Functional Test Supplier - Invite a contributor feature'
+  When I enter 'simulate-delivered@notifications.service.gov.uk' in the 'Email address' field
```

Comment:
Can we stick to the "safe" domains mentioned in https://alphagov.github.io/digitalmarketplace-manual/emails.html#live-stages ? It just makes all our stuff less notify-specific (and also emails to `simulate-delivered@notifications.service.gov.uk` don't actually show up in the "sent" log so there's a limit to how much we can test things against them (I know that doesn't make a difference in this specific case)

Reply:
Will do